lightning/src/ln/channelmanager.rs

```diff
@@ -5586,20 +5697,13 @@ mod tests {
 		nodes[0].node.handle_revoke_and_ack(&nodes[1].node.get_our_node_id(), &bs_third_raa);
 		check_added_monitors!(nodes[0], 1);
 
-		// There's an existing bug that generates a PaymentSent event for each MPP path, so handle that here.
```

Comment:
nit: Is it worth having a comment here that explicitly calls out that we only get one event here?